media/css/firefox/whatsnew/whatsnew-67.scss

```diff
@@ -0,0 +1,500 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+$font-path: '/media/fonts';
+$image-path: '/media/protocol/img';
+
+@import '../../../protocol/css/includes/lib';
+@import '../../../protocol/css/components/call-out';
+@import '../../../protocol/css/components/feature-card';
+@import '../../../protocol/css/components/hero';
+@import '../../../protocol/css/components/modal';
+@import '../../../protocol/css/components/newsletter-form';
+
+@import '../../protocol/components/state-fxa';
+
+
+main {
+    color: $color-off-black;
+}
+
+//* -------------------------------------------------------------------------- */
+// Up-to-date page header
+
+.c-page-header-up-to-date {
+    @include text-display-xxs;
+    display: none;
+
+    span {
+        background-color: #6EE7C4;
+        border-radius: 2px;
+        display: inline-block;
+        font-weight: normal;
+        padding: $spacing-sm $spacing-md;
+        max-width: $content-sm;
+    }
+
+    .is-up-to-date & {
+        display: block;
+    }
+}
+
+@media #{$mq-md} {
+    .c-page-header-inner {
+        align-items: center;
+        display: grid;
+        grid-template-columns: 1fr max-content 1fr;
+    }
+
+    .c-page-header-up-to-date {
+        margin: 0;
+    }
+}
+
+
+
+//* -------------------------------------------------------------------------- */
+// Intro section
+
+.c-page-title {
+    @include text-display-xl;
+    max-width: $content-md;
+    margin: 0 auto $layout-sm auto;
+    text-align: center;
+}
+
+.mzp-c-card-feature.mzp-l-card-feature-right-66.wn67-intro {
+    margin-bottom: $layout-md;
+    max-width: $content-lg;
+
+    .mzp-c-card-feature-title {
+        @include visually-hidden;
```

Comment:
Curious why this title is hidden visually but not from screen readers. Would it be worth adding a comment to explain?

Reply:
I thought I needed to preserve the heading order but on closer inspection it's not necessary.